includes/admin/class-wc-stripe-onboarding-controller.php

```diff
@@ -73,9 +73,10 @@ public function admin_scripts() {
 	 * Create an admin page without a side menu: wp-admin/admin.php?page=wc_stripe-onboarding_wizard
 	 */
 	public function add_onboarding_wizard() {
+		// This submenu is hidden from the admin menu
 		add_submenu_page(
-			null, // Hide this submenu from admin menu
-			__( 'Onboarding Wizard', 'woocommerce-gateway-stripe' ),
+			'admin.php',
```

Comment:
Nice one! Although, isn't this a bit of a hack? Won't using `'admin.php'` won't have any unintended side effects?

Reply:
I thought `null` was a bit of a hack :D I tested this in both the "new" navigation and the "normal" menus, I couldn't find a difference in the behavior compared to using `null` (besides the `title` not appearing when `null` was used)

Reply:
> I thought null was a bit of a hack :D

True! It shouldn't be an issue, I was just thinking if there can be any unintended consequences, like if other plugins also use `admin.php` or something. Anyways, I highly doubt this will be problematic, especially with the testing you did!